tools/flaky-test-reporter/result.go

```diff
@@ -88,18 +88,23 @@ func (ts *TestStat) getTestStatus() string {
 	}
 }
 
-func getFlakyRate(testStats map[string]*TestStat) (float32, error) {
-	totalCount := len(testStats)
-	if 0 == totalCount {
-		return 0.0, nil
-	}
-	flakyCount := 0
-	for _, ts := range testStats {
+func getFlakyTests(rd *RepoData) []string {
+	var flakyTests []string
+	for testName, ts := range rd.TestStats {
 		if ts.isFlaky() {
-			flakyCount++
+			flakyTests = append(flakyTests, testName)
 		}
 	}
-	return float32(flakyCount)/float32(totalCount), nil
+	return flakyTests
+}
+
+
+func getFlakyRate(rd *RepoData) float32 {
```

Comment:
any reason this is float32 and not float64?

Reply:
This rate is used to be compared with a threshold like 1%, so the precision of float32 is sufficient